spec/models/tasks/judge_task_spec.rb

```diff
@@ -11,6 +11,63 @@
     create(:staff, :attorney_role, sdomainid: attorney.css_id)
   end
 
+  describe "only_open_task_of_type" do
+    let(:appeal) { create(:appeal) }
+    let!(:first_assign_task) do
+      create(:ama_judge_assign_task, assigned_to: judge, appeal: appeal)
+    end
+    let(:second_assign_task) do
+      create(:ama_judge_assign_task, assigned_to: judge, appeal: appeal)
+    end
+
+    subject { second_assign_task }
+
+    context "only one judge assign task can be created for an appeal" do
+      it "throws an error when a second task is created" do
+        expect { subject }.to raise_error do |error|
+          expect(error).to be_a(ActiveRecord::RecordInvalid)
+        end
+      end
+    end
```

Comment:
Less indirection and more readable:
```suggestion
    context "when one judge assign task is open" do
      it "throws an error when a second judge assign task is created" do
        expect { create(:ama_judge_assign_task, assigned_to: judge, appeal: appeal) }.to raise_error do |error|
          expect(error).to be_a(ActiveRecord::RecordInvalid)
        end
      end
    end
```